src/extensions/extensions-store.ts

```diff
@@ -61,7 +51,10 @@ export class ExtensionsStore extends BaseStore<LensExtensionsStoreModel> {
 
   isEnabled(extId: LensExtensionId) {
     const state = this.state.get(extId);
-    return state && state.enabled; // by default false
+
+    // By default false, so that copied extensions are disabled by default.
+    // If user installs the extension from the UI, the Extensions component will specifically enable it.
+    return Boolean(state && state.enabled);
```

Comment:
```suggestion
    return !!state?.enabled;
```

Reply:
I think `Boolean` is more readable than using `!!`. I think `?.` / `&&` are equivalent in readability.

Reply:
I a sort of moot on `Boolean` since it doesn't seem to be used much. I quite prefer `?.` over `&&` as the later can get very long quite quickly.

Reply:
I converted to `?.` but kept `Boolean` for now. It's true some don't know it, but I think it's pretty standard JavaScript, and every project is an opportunity to make people familiar with it. Quick look at Google seems to favor `Boolean` over `!!`.